### PR TITLE
Feat/674070 tfp | cleanup logic for old ske credentials rotation flow

### PR DESCRIPTION
**Pull request by bahkauv70**
PR to address issue https://jira.schwarz/browse/STACKITTPR-65. 
This removes references to the read-only _kube_config_ attribute in the cluster definition

### CODE REVIEW

File: stackit/internal/services/ske/cluster/datasource.go [hunk 1]
```diff
@@ -16,7 +15,6 @@ import (
 	"github.com/stackitcloud/stackit-sdk-go/services/ske"
 	"github.com/stackitcloud/terraform-provider-stackit/stackit/internal/core"
 	"github.com/stackitcloud/terraform-provider-stackit/stackit/internal/validate"
-	"golang.org/x/mod/semver"
 )
 
 // Ensure the implementation satisfies the expected interfaces.
```
Comment (vicentepinto98) [inline stackit/internal/services/ske/cluster/datasource.go:20]: Also remove the field `kube_config` from the schema, on line 318

File: docs/data-sources/resourcemanager_project.md [hunk 2]
```diff
@@ -33,5 +32,16 @@ data "stackit_resourcemanager_project" "example" {
 
 - `id` (String) Terraform's internal data source. ID. It is structured as "`container_id`".
 - `labels` (Map of String) Labels are key-value string pairs which can be attached to a resource container. A label key must match the regex [A-ZÄÜÖa-zäüöß0-9_-]{1,64}. A label value must match the regex ^$|[A-ZÄÜÖa-zäüöß0-9_-]{1,64}
+- `members` (Attributes List, Deprecated) The members assigned to the project. At least one subject needs to be a user, and not a client or service account. This value is only considered during creation. Changing it afterwards will have no effect.
```
Comment (vicentepinto98) [inline docs/data-sources/resourcemanager_project.md:35]: Please delete this file, currently we don't want to show the `members` field in the documentation, as it was a feature that we introduced but had to rollback. So for now we always need to remove it manually when it is generated automatically

File: docs/resources/key_pair.md [hunk 3]
```diff
@@ -5,24 +5,26 @@ subcategory: ""
 description: |-
   Key pair resource schema. Must have a region specified in the provider configuration. Allows uploading an SSH public key to be used for server authentication.
   Usage with server
-  ```terraform
-  resource "stackitkeypair" "keypair" {
+
```
Comment (vicentepinto98) [inline docs/resources/key_pair.md:8]: unrelated changes, not sure if they were missed in a previous PR or if you didn't merge with main

File: docs/resources/server.md [hunk 4]
```diff
@@ -7,43 +7,46 @@ description: |-
   ~> This resource is in beta and may be subject to breaking changes in the future. Use with caution. See our guide https://registry.terraform.io/providers/stackitcloud/stackit/latest/docs/guides/opting_into_beta_resources for how to opt-in to use beta resources.
   Example Usage
   With key pair
-  ```terraform
-  resource "stackitkeypair" "keypair" {
+
```
Comment (vicentepinto98) [inline docs/resources/server.md:10]: unrelated changes, not sure if they were missed in a previous PR or if you didn't merge with main

File: docs/resources/resourcemanager_project.md [hunk 5]
```diff
@@ -35,9 +35,20 @@ resource "stackit_resourcemanager_project" "example" {
 ### Optional
 
 - `labels` (Map of String) Labels are key-value string pairs which can be attached to a resource container. A label key must match the regex [A-ZÄÜÖa-zäüöß0-9_-]{1,64}. A label value must match the regex ^$|[A-ZÄÜÖa-zäüöß0-9_-]{1,64}
+- `members` (Attributes List, Deprecated) The members assigned to the project. At least one subject needs to be a user, and not a client or service account. This value is only considered during creation. Changing it afterwards will have no effect.
```
Comment (vicentepinto98) [inline docs/resources/resourcemanager_project.md:38]: same here, this file should not be updated

File: docs/resources/key_pair.md (hunk 3 above)
Comment (bahkauv70) [reply docs/resources/key_pair.md:8]: This change was created by the doc generation. I'll inspect how this happened

File: docs/resources/resourcemanager_project.md (hunk 5 above)
Comment (bahkauv70) [reply docs/resources/resourcemanager_project.md:38]: This change was created by the doc generation. I'll inspect how this happened

Comment (vicentepinto98) [reply docs/resources/resourcemanager_project.md:38]: I is expected, but we should delete it manually. Like described here: https://github.com/stackitcloud/terraform-provider-stackit/pull/611#discussion_r1887169437

Comment (bahkauv70) [reply docs/resources/resourcemanager_project.md:38]: As discussed, unrelated change caused by missing doc generation on main, but will be kept to cleanup main as well.

File: docs/resources/server.md (hunk 4 above)
Comment (bahkauv70) [reply docs/resources/server.md:10]: As discussed, unrelated change caused by missing doc generation on main, but will be kept to cleanup main as well.

File: docs/data-sources/resourcemanager_project.md (hunk 2 above)
Comment (vicentepinto98) [reply docs/data-sources/resourcemanager_project.md:35]: only the change

Comment (vicentepinto98) [reply docs/data-sources/resourcemanager_project.md:35]: on both datasource and resource folders

Comment (bahkauv70) [reply docs/data-sources/resourcemanager_project.md:35]: removed members section

File: docs/resources/resourcemanager_project.md (hunk 5 above)
Comment (bahkauv70) [reply docs/resources/resourcemanager_project.md:38]: removed members section

File: docs/resources/resourcemanager_project.md [hunk 6]
```diff
@@ -41,3 +41,11 @@ resource "stackit_resourcemanager_project" "example" {
 - `container_id` (String) Project container ID. Globally unique, user-friendly identifier.
 - `id` (String) Terraform's internal resource ID. It is structured as "`container_id`".
 - `project_id` (String) Project UUID identifier. This is the ID that can be used in most of the other resources to identify the project.
+
```
Comment (vicentepinto98) [inline docs/resources/resourcemanager_project.md:44]: this should also be removed

File: docs/data-sources/resourcemanager_project.md [hunk 7]
```diff
@@ -35,3 +34,11 @@ data "stackit_resourcemanager_project" "example" {
 - `labels` (Map of String) Labels are key-value string pairs which can be attached to a resource container. A label key must match the regex [A-ZÄÜÖa-zäüöß0-9_-]{1,64}. A label value must match the regex ^$|[A-ZÄÜÖa-zäüöß0-9_-]{1,64}
 - `name` (String) Project name.
 - `parent_container_id` (String) Parent resource identifier. Both container ID (user-friendly) and UUID are supported
+
```
Comment (vicentepinto98) [inline docs/data-sources/resourcemanager_project.md:37]: this should also be removed

File: docs/resources/resourcemanager_project.md (hunk 6 above)
Comment (bahkauv70) [reply docs/resources/resourcemanager_project.md:44]: removed block

File: docs/data-sources/resourcemanager_project.md (hunk 7 above)
Comment (bahkauv70) [reply docs/data-sources/resourcemanager_project.md:37]: removed block

**Review by vicentepinto98 — APPROVED** [review]
